C++/C.sublime-syntax

```diff
@@ -122,12 +128,45 @@ contexts:
     - match: \b({{control_keywords}})\b
       scope: keyword.control.c
     - match: \bsizeof\b
-      scope: keyword.operator.word.c
+      scope: keyword.operator.word.c debug.keywords
 
   modifiers:
     - match: \b({{modifiers}})\b
       scope: storage.modifier.c
 
+  modifiers-macros:
+    - match: |-
+        (?x:
+          (?:{{macro_identifier}}|{{attr_macro_identifier}})
+          (?=
+            \(
+            \s*
+            (?>
```

Comment:
Atomic groups are not supported by ST's sregex engine and cause the slower Oniguruma to be triggered. Is there a certain reason to use them?

Reply:
And here I thought I was making things faster. I'll remove them.